pandoc.cabal

```diff
@@ -596,6 +597,8 @@ library
                    Text.Pandoc.Readers.EPUB,
                    Text.Pandoc.Readers.Muse,
                    Text.Pandoc.Readers.Man,
+                   Text.Pandoc.Readers.Mdoc,
+                   Text.Pandoc.Readers.Mdoc.Lex,
```

Comment:
Does the Mdoc.Lex module need to be part of the public interface?

Reply:
Nope, leftover from development. Fixed.